tomviz/python/AutoTiltAxisShiftAlignment.py

```diff
@@ -32,17 +32,18 @@ def transform_scalars(self, dataset):
         I = np.zeros(shifts.size)
 
         self.progress.maximum = shifts.size - 1
-        step = 0
+        step = 1
+        total = shifts.size-1
 
         for i in range(shifts.size):
             if self.canceled:
                 return
             shiftedTiltSeries = np.roll(
                 tiltSeries[slices, :, :, ], shifts[i], axis=1)
             for s in range(numberOfSlices):
-                self.progress.message = ('Reconstructing slice No.%d with %d '
+                self.progress.message = ('Reconstructing slice No.%d/%d with %d'
                                          'pixels shift' %
-                                         (slices[s], shifts[i]))
+                                         (step, total, shifts[i]))
```

Comment:
This one seems to have changed.  The old behavior printed out something like this:
```
"Reconstructing slice No.97 with 20 pixels shift"
"Reconstructing slice No.1 with 20 pixels shift"
"Reconstructing slice No.19 with 20 pixels shift"
"Reconstructing slice No.95 with 20 pixels shift"
"Reconstructing slice No.23 with 20 pixels shift"
```
For each shift from -20 to 20.

The new one prints out
```
"Reconstructing slice No.1/40 with -20 pixels shift"
"Reconstructing slice No.1/40 with -20 pixels shift"
"Reconstructing slice No.1/40 with -20 pixels shift"
"Reconstructing slice No.1/40 with -20 pixels shift"
"Reconstructing slice No.1/40 with -20 pixels shift"
```
So we seem to have lost information.  Maybe it should just print out once, something like `Reconstructing slices for shift X`.

Reply:
Background: I changed the progress handler to print out the progress messages so I could see them in order.

Reply:
@jtschwar how much detail do we want to give in the progress messages.  If there is too much, then noone will be able to read it since it will flash by too quick.  Would it make more sense to back off to 'Reconstructing slices for shift -20' or do we think the user cares about seeing information on every reconstruction?